src/viewport.cpp

```diff
@@ -979,6 +979,67 @@ static void DrawAutorailSelection(const TileInfo *ti, uint autorail_type)
 	DrawSelectionSprite(image, _thd.make_square_red ? PALETTE_SEL_TILE_RED : pal, ti, 7, foundation_part);
 }
 
+enum TileHighlightType {
+	THT_NONE,
+	THT_WHITE,
+	THT_BLUE,
+	THT_RED,
+};
+
+const Station *_station_th; ///< Currently selected station for coverage area highlight
+const Town *_town_th;       ///< Currently selected town for coverage area highlight
```

Comment:
These names definitely made me "huh?" for a while until I discovered th = tile highlight. Can I suggest `_viewport_highlight_town` or similar?

Reply:
Done